### PR TITLE
feat: create component for rendering page title for use cases page

### PR DESCRIPTION
**Pull request by hacktivist123**
**This PR:**

- adds a new component for handling page titles in the use case pages
- Updates all the use cases pages to use the component

**Steps to test**
- Open any of the [pages](https://deploy-preview-309--cilium.netlify.app/use-cases/cluster-mesh/) in the use case section
- See the title on the tab, it must match the title of the page

**Before this PR**

Screenshot 2023-07-10 at 22 11 18

**After this PR**

Screenshot 2023-07-10 at 22 12 21

### CODE REVIEW

**Comment by vannyle** [comment]
Hi @xmulligan @hacktivist123, I believe you can add metadata for pages by using the [gatsby Head function](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-head/) instead of `react-helmet`. Please see the example [here](https://github.com/cilium/cilium.io/blob/main/src/pages/brand.jsx#L25-L27).

**Comment by hacktivist123** [comment]
Hello @vannyle you're absolutely correct and it looks like I over-engineered the solution, thanks for catching that. I have not written Gatsby in a while so I forgot Gatsby has this inbuilt already. I have gone ahead to make the required changes in my most recent PR, kindly let me know what you think.

File: src/pages/use-cases/kube-proxy.jsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import PageTitle from 'components/pages/use-cases/page-title';
```
Comment (vannyle) [inline src/pages/use-cases/kube-proxy.jsx:1]: LGTM! Only one thing that you forgot to remove the PageTitle, that's why the build failed.

**Review by vannyle — APPROVED** [review]

File: src/pages/use-cases/kube-proxy.jsx (hunk 1 above)
Comment (hacktivist123) [reply src/pages/use-cases/kube-proxy.jsx]: yep, just updated it

**Review by xmulligan — APPROVED** [review]

**Pull request merged**